app/src/main/res/xml/preferences_notification.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <SwitchPreferenceCompat
+        android:defaultValue="true"
+        android:enabled="true"
+        android:key="prefShowDownloadReport"
+        android:summary="@string/pref_showDownloadReport_sum"
+        android:title="@string/pref_showDownloadReport_title" />
+    <SwitchPreferenceCompat
+        android:defaultValue="false"
+        android:enabled="true"
+        android:key="prefShowAutoDownloadReport"
+        android:summary="@string/pref_showAutoDownloadReport_sum"
+        android:title="@string/pref_showAutoDownloadReport_title" />
+    <SwitchPreferenceCompat
+        android:defaultValue="true"
+        android:key="pref_gpodnet_notifications"
+        android:summary="@string/pref_gpodnet_notifications_sum"
+        android:title="@string/pref_gpodnet_notifications_title" />
```

Comment:
I think some of the titles can be simplified, now that it is its own screen. Instead of 

```
== Notifications ==
Show download report
Show auto download report
Show notifications on sync errors
```

I would suggest something like

```
== Notifications ==
Automatic download completed
Download errors
Synchronization errors
```

@keunes thoughts?

Reply:
Good idea. Thinking about it for a while, I think it'd be good to unify their form (include the verb). And maybe add an explanatory text above, to give some context? (just extended the list to show how it could work when more notification options are added)

<pre>
== Notifications ==

Show a notification for the following events:

new episode available (-> open new screen with 'all' switch & list of per podcast-settings below)
automatic download completed
download (of what?) failed
synchronization failed
automatic back-up failed
</pre>

Reply:
> download (of what?) failed

Episodes or feeds. The same notification is used for both.

Reply:
And what do you think of adding that extra sentence? Or is something like that already in place?

Reply:
It's not in place but starting with Android 8.0, the system manages notification settings like those. I wouldn't spend too much time on the UI here because I want to rather show the system settings and use that screen as a fall-back for older devices. System settings have a lot more options than just enable/disable.